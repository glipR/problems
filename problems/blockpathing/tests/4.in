30000000 300000000
392000000 200000000
50000000 50000000 300000000 500000000
40000000 350000000 200000000
