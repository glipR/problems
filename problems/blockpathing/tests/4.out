677282254.6224204
